instrumentation/opentelemetry-instrumentation-aiopg/pyproject.toml

```diff
@@ -33,7 +33,7 @@ dependencies = [
 
 [project.optional-dependencies]
 instruments = [
-  "aiopg >= 0.13.0, < 1.3.0",
+  "aiopg >= 0.13.0, <= 2.0.0",
```

Comment:
Sorry for the earlier comment, which included `=`. This should be < 2.0.0 since 2.x can included breaking changes